packages/core-components/api-report.md

```diff
@@ -1447,6 +1473,12 @@ export class UserIdentity implements IdentityApi {
   signOut(): Promise<void>;
 }
 
+// @public
+export const useSidebarOpenState: () => SidebarOpenState;
```

Comment:
Yeah, I can't really speak to the underlying reasoning/logic of either of these contexts.  There are definitely interdependencies between them (e.g. as far as I can understand, `isOpen` will always be `true` when `isPinned` is `true`), and it's unclear how `isMobile` relates).

...It could be beneficial to refactor.  ...But I wanted to focus on the package-specific state bug first.

Reply:
Agree, thanks for explaining!